<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration_RANSAC</class>
 <widget class="QWidget" name="Registration_RANSAC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Correspondences Rejection</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="Registration_CorrespondenceRejection" name="myCorrespondenceRejection" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="PointClouds">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>79</height>
      </size>
     </property>
     <property name="title">
      <string>Point Clouds</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_Model">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_PC">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_ApplyOldTranslation">
        <property name="text">
         <string>ApplyOldT</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>PC</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton_ApplyOldOrientation">
        <property name="text">
         <string>ApplyOldR</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="pushButton_UseNextPair">
        <property name="text">
         <string>Next pair</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Downsample</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="Filters_Downsampling" name="myDownsampling" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Correspondences Estimation</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="Registration_CorrespondenceEstimation" name="myCorrespondenceEstimation" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Description</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="Description_Keypoints" name="myDescriptor" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>RANSAC</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_Times">
        <property name="text">
         <string>Save Times</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QRadioButton" name="Registration_TranslationOnly">
        <property name="text">
         <string>TranslationOnly</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">Registration_ButtonGroup_TransformationEstimation</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_Metrics">
        <property name="text">
         <string>Save Metrics</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="Registration_SVD">
        <property name="text">
         <string>SVD</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">Registration_ButtonGroup_TransformationEstimation</string>
        </attribute>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QCheckBox" name="checkBox_Refine">
        <property name="text">
         <string>Refine</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_Registration_RANSAC3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RegRANSAC3</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_Registration_RANSAC">
        <property name="text">
         <string>RegRANSAC</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QDoubleSpinBox" name="Registration_SAC_InlierThreshold">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="maximum">
         <double>99999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.005000000000000</double>
        </property>
        <property name="value">
         <double>0.020000000000000</double>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSpinBox" name="Registration_SAC_MaxIteration">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="singleStep">
         <number>10</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Inlier Threshold</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max Iteration</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_DistThresh">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_Calc_Dist">
        <property name="text">
         <string>Calulate Distances</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Registration_CorrespondenceEstimation</class>
   <extends>QWidget</extends>
   <header>Elaboration/registration_correspondenceestimation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Registration_CorrespondenceRejection</class>
   <extends>QWidget</extends>
   <header>Elaboration/registration_correspondencerejection.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Filters_Downsampling</class>
   <extends>QWidget</extends>
   <header>Elaboration/filters_downsampling.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Description_Keypoints</class>
   <extends>QWidget</extends>
   <header>Elaboration/description_keypoints.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pushButton_Calc_Dist</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="Registration_ButtonGroup_TransformationEstimation"/>
 </buttongroups>
</ui>
